### PR TITLE
add heroku tutorial

### PR DESCRIPTION
**Pull request by sw-yx**
as requested in #463

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=468) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=468) before we can accept your contribution.<br/><hr/>**sw-yx** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-nodejs?pullRequest=468) it.</sub>

**Comment by sw-yx** [comment]
signed license agreement

**Comment by mbernier** [comment]
@sw-yx Can you please look into the conflicts and merge them? Sorry we haven't gotten to this just yet - we are moving through everything as quickly as possible!

**Comment by mbernier** [comment]
@sw-yx

We have not been able to merge your Pull Request, but because you are awesome - we wanted to make sure you could still get a SendGrid Hacktoberfest shirt.  

Please go **fill [out our swag form](https://dx.sendgrid.com/swag) before Nov 5th** and we will send the shirt! (We know that you might have tried this before and it didn’t work, sorry about that!)

### **You have till Nov 5th to fill out this form** in order to get the Hacktoberfest shirt!

Thank you for contributing during Hacktoberfest! We hope to see you in the repos soon! Just so you know, we always give away a SendGrid shirt for your first ever non-Hacktoberfest PR that gets merged.

**Comment by sw-yx** [comment]
ok sure. I think i resolved all conflicts btw!

**Comment by thinkingserious** [comment]
Nice, thanks!

File: package-lock.json [hunk 1]
```diff
@@ -5546,6 +5546,12 @@
       "integrity": "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=",
       "dev": true
     },
+    "typescript": {
```
Comment (thinkingserious) [inline package-lock.json]: Not sure why this was added to this commit. Was this a mistake?

**Comment by thinkingserious** [comment]
@sw-yx,

Could you please review my requested changes so that I can get this merged? :)

Thanks!

With Best Regards,

Elmer

**Comment by sw-yx** [comment]
Hey Elmer sorry for the delay in reply. holiday :) i was a noobie back then and commited the package lock change. i have undone that now, thank you.

**Comment by thinkingserious** [comment]
Hello @sw-yx,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**